app/simplified-account-routes.js

```diff
@@ -38,7 +38,7 @@ simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.refund
 simplifiedAccount.post(paths.simplifiedAccount.settings.emailNotifications.refundEmailToggle, enforceEmailCollectionModeNotOff, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.refundEmails.post)
 simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.paymentConfirmationEmailToggle, enforceEmailCollectionModeNotOff, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.paymentConfirmationEmails.get)
 simplifiedAccount.post(paths.simplifiedAccount.settings.emailNotifications.paymentConfirmationEmailToggle, enforceEmailCollectionModeNotOff, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.paymentConfirmationEmails.post)
-simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.templates, permission('email-notification-template:read'), serviceSettingsController.emailNotifications.templates.get)
+simplifiedAccount.get(paths.simplifiedAccount.settings.emailNotifications.templates, permission('transactions:read'), serviceSettingsController.emailNotifications.templates.get)
```

Comment:
It does exist, but the way it currently is, a non-admin user wouldn't be able to access the templates page on that permission. The ['transactions:read' permission is on the email notifications index page](https://github.com/alphagov/pay-selfservice/blob/master/app/simplified-account-routes.js#L34) so non-admins can access that page, so it should be the same here. 
It's possible this is improperly defined, but fixing that would require a lot of work as it touches the database.

Reply:
I would strongly favour fixing the role definitions rather than hacking around it, although I appreciate that's not an easy task. Since all this work is still WIP would it not be better to code in the proper permissions and write a ticket for sorting out the roles?

Reply:
I can certainly write a police-cat ticket for someone to delve further into it, but I don't see this as a problem just now. I only said it's possible it is improperly defined, but I'm not 100% that's true. For non-admin users, we do want them to be able to the [email notifications](https://www.figma.com/design/2xrNzEL8AjblsiPoS3cAY9/GOV.UK-Pay-design-work?node-id=8142-33063&node-type=frame&t=J1WRxIf34Y24tSWx-0) and [email templates](https://www.figma.com/design/2xrNzEL8AjblsiPoS3cAY9/GOV.UK-Pay-design-work?node-id=9193-26204&node-type=frame&t=J1WRxIf34Y24tSWx-0) screens, and you can see there is a banner informing them they don't have permissions to edit things.
So I think it's all ok for now and we don't need to sort this out now for a minimum viable product.

Reply:
And although permissions are defined at a granular level, they're mostly assigned to the admin role. That granularity isn't something we utilise extensively at the moment.

Reply:
Ok it looks like having 'email-notification-template:read' already works for real as that permission is assigned to view-only roles. It's my cypress test that needs amending.